zim/gui/uiactions.py

```diff
@@ -272,7 +272,7 @@ def show_templateeditor(self):
 		from zim.gui.templateeditordialog import TemplateEditorDialog
 		TemplateEditorDialog(self.widget).run()
 
-	@action(_('Pr_eferences')) # T: Menu item
+	@action(_('Pr_eferences'), '<Primary>comma') # T: Menu item
```

Comment:
You might not want this accelerator for platforms other than macOS. I added the accelerator there because I didn't manage te add it in the macOS Menubar plugin.